frontend/src/metabase-lib/drills.unit.spec.ts

```diff
@@ -37,3 +40,32 @@ describe("availableDrillThrus", () => {
     ]);
   });
 });
+
+describe("drillThru", () => {
+  it("should apply sort drill", () => {
+    const query = createQuery();
+    const stageIndex = -1;
+    const columns = orderableColumns(query, stageIndex);
+    const column = columnFinder(query, columns)("ORDERS", "SUBTOTAL");
+
+    const drills = availableDrillThrus(
+      query,
+      stageIndex,
+      column,
+      undefined,
+      null,
+      null,
+    );
+
+    const sortDrill = checkNotNull(
+      drills.find(
+        drill =>
+          displayInfo(query, stageIndex, drill)?.type === "drill-thru/sort",
+      ),
+    );
```

Comment:
WDYT about just throwing an error if `sortDrill` is `undefined`? I think we should try to avoid imports from `metabase/*` here. I can see how `checkNotNull` is more useful here, perhaps we can figure out another way eventually